### ISSUE TITLE
`auro-combobox` a11y mobile: combobox modal popover allows pass through touch

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-formkit#691]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [x] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [x] Mobile

### Steps to reproduce

For repro steps, see the attached video. iPhone iOS 18.5

https://github.com/user-attachments/assets/e82600d2-265f-4c1f-9d39-f5fdce876f0e

Notice in the video, a user can successfully move through most of the interactive elements easily. The issue comes up when a user may need to edit the contents of a selection. Moving between the keyboard and the insertion point of the editable text, a user must tap somewhere in the visible screen. 

It is at this point a user is actually able to select items BEHIND the modal UI. Once a user has tapped into the visible screen space and then uses the swipe gesture to move through the content in the UI, VoiceOver starts to read aloud content that is not visible or relevant to the actions the user is trying to address. 

This will be very confusing to a user with assistive technology as the context of their actions is incorrect as to their task. 

### Expected behavior

It would be expected that the modal would restrict a user's ability to only select things that are within the viewable space and applicable to the task at hand. 

### Which version of auro-formkit do you have installed?

@latest

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by jason-capsule42** [comment]
This work is no longer necessary because it is handled by the new auro-layover functionality to be released soon.
